src/parse.c

```diff
@@ -3539,7 +3541,7 @@ process_document(NetplanParser* npp, GError** error)
 {
     gboolean ret;
     int previously_found;
-    int still_missing;
+    guint still_missing;
```

Comment:
note: this is due to `g_hash_table_size` returning a `guint`. For consistency's sake, we might transform `previously_found` and other related numbers to `guint`, too. WDYT?